src/mca/schizo/ompi/schizo_ompi.c

```diff
@@ -1848,13 +1853,15 @@ static void allow_run_as_root(prte_cli_result_t *results)
     /* we always run last */
     char *r1, *r2;
 
-    if (prte_cmd_line_is_taken(results, "allow-run-as-root")) {
+    if (pmix_cmd_line_is_taken(results, "allow-run-as-root")) {
```

Comment:
This slipped thru - should be "prte_cmd_line_is_taken" in this older branch. Not sure how it managed to compile 🤷‍♂️